docs/spec/qualifiers.rst

```diff
@@ -272,33 +283,54 @@ details of the syntax:
     SmallInt = Annotated[int, ValueRange(0, 100)]
     SmallInt(1)  # Type error
 
+:pep:`593` and an earlier version of this specification used the term
+"annotations" instead of "metadata" for the extra arguments to
+``Annotated``. The term "annotations" is deprecated to avoid confusion
```

Comment:
I find it a bit odd to say that the "term is deprecated". The typing spec is the source of truth here, so is it necessary to clarify that it deviates in terminology from the PEP or from earlier versions of the spec? We don't do this elsewhere in the spec where we have clarified or updated terminology.

Reply:
I think it's helpful in cases where another term was previously in use. I'm not sure there is any comparable situation elsewhere in the spec.